File: iOS/VoiceRoomKit/NEGame/NEGameKit/NEGameKit/Classes/Public/Common/NEGameKitOptions.swift
// Copyright (c) 2022 NetEase, Inc. All rights reserved.
// Use of this source code is governed by a MIT license that can be
// found in the LICENSE file.

import Foundation

@objcMembers

/// NEGameKit 配置项
public class NEGameKitOptions: NSObject {
  /// appKey 为roomkit的Key
  public var appKey: String = ""
  /// 房间ID
  public var roomUuid: String = ""
  /// 服务器地址
  public var baseUrl: String?
}

File: Android/gamekit/gamekit/src/main/java/com/netease/yunxin/kit/gamekit/api/model/NEGameKitOptions.kt
/*
 * Copyright (c) 2022 NetEase, Inc. All rights reserved.
 * Use of this source code is governed by a MIT license that can be
 * found in the LICENSE file.
 */

package com.netease.yunxin.kit.gamekit.api.model

/**
 * 提供初始化配置
 * @property appKey NERoom appKey
 * @property roomUuid 房间Id
 * @property baseUrl 服务器地址
 */
data class NEGameKitOptions(
    val appKey: String,
    val roomUuid: String,
    val baseUrl: String
)
